voltaire_bundler/cli_manager.py

```
@@ -1055,6 +1105,61 @@ async def get_init_data(args: Namespace) -> InitData:
+        if args.gas_price_refresh_interval is not None
+        else default_refresh_interval(args.chain_id)
+    )
+    gas_price_cache = GasPriceCache(
+        ethereum_node_urls_rearranged,
+        args.chain_id,
+        args.legacy_mode,
+        gas_price_refresh_interval,
+    )
+    try:
+        await gas_price_cache.warm()
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)